cmd/crowdsec-cli/config.go

```diff
@@ -41,14 +25,13 @@ If no commands are specified, config is in interactive mode.`,
 		Long:  `Displays the current cli configuration.`,
 		Args:  cobra.ExactArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
-			if config.output == "json" {
+			if csConfig.Cscli.Output == "json" {
```

Comment:
yes csConfig is a global that is instantiated in main, as it's used in callbacks such as `initConfig` for `cobra.OnInitialize()` I'd figure it would be simpler. An alternative would be to redeclare it in every `Run:` func ?